src/main/java/fr/cnes/sonarqube/plugins/framac/rules/FramaCRulesDefinition.java

```diff
@@ -1,41 +1,35 @@
 /*
-	 * This file is part of sonar-frama-c-plugin.
-	 *
-	 * sonar-frama-c-plugin is free software: you can redistribute it and/or modify
-	 * it under the terms of the GNU General Public License as published by
-	 * the Free Software Foundation, either version 3 of the License, or
-	 * (at your option) any later version.
-	 *
-	 * sonar-frama-c-plugin is distributed in the hope that it will be useful,
-	 * but WITHOUT ANY WARRANTY; without even the implied warranty of
-	 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-	 * GNU General Public License for more details.
-	 *
-	 * You should have received a copy of the GNU General Public License
-	 * along with sonar-frama-c-plugin.  If not, see <http://www.gnu.org/licenses/>.
-
-*/
-
+ * This file is part of sonarframac.
+ *
+ * sonarframac is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sonarframac is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with sonarframac.  If not, see <http://www.gnu.org/licenses/>.
+ */
 package fr.cnes.sonarqube.plugins.framac.rules;
 
-import java.io.InputStream;
-import java.nio.charset.StandardCharsets;
-
+import fr.cnes.sonarqube.plugins.framac.languages.FramaCLanguage;
 import org.sonar.api.server.rule.RulesDefinition;
 import org.sonar.api.server.rule.RulesDefinitionXmlLoader;
 
-import fr.cnes.sonarqube.plugins.framac.languages.FramaCLanguage;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
 
 /**
- * Define specific Frama-C rules from a given resource file: { @link
+ * Define specific Frama-C definedRules from a given resource file: { @link
  * FramaCRulesDefinition#PATH_TO_RULES_XML }
- * 
- * @author Cyrille FRANCOIS
- *
  */
 public class FramaCRulesDefinition implements RulesDefinition {
 
-	public static final String PATH_TO_RULES_XML = "/default/framac-rules.xml";
+	public static final String PATH_TO_RULES_XML = "/rules/framac-rules.xml";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor your code to get this URI from a customizable parameter. ([squid:S1075](https://sonarcloud.io/organizations/lequal/rules#rule_key=squid%3AS1075))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fr.cnes.sonarqube.plugins%3Asonar-frama-c-plugin&pullRequest=20&issues=AWXJbamWashydt7kyasA&open=AWXJbamWashydt7kyasA)

[ISSUE_KEY]: AWXJbamWashydt7kyasA